src/Fixer/Casing/MagicMethodCasingFixer.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Casing;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Analyzer\ArgumentsAnalyzer;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class MagicMethodCasingFixer extends AbstractFixer
+{
+    private static $magicNames = [
+        '__call' => [
+            'name' => '__call',
+            'argumentsRange' => [2, 2],
+        ],
+        '__callstatic' => [
+            'name' => '__callStatic',
+            'argumentsRange' => [2, 2],
```

Comment:
why is casing related to number of params ?

Reply:
If the number of arguments do not match the signature as defined as magic PHP method it does not get fixed as it assumed as overridden, to be a safe-ish side, you think we could do without? happy to remove